contracts/Oracle.sol

```diff
@@ -221,8 +221,12 @@ contract Oracle is Ownable {
         require(!isDisputePeriodOver(_asset, _expiryTimestamp), "Oracle: dispute period over");
 
         Price storage priceToUpdate = storedPrice[_asset][_expiryTimestamp];
+
+        require(priceToUpdate.timestamp != 0, "Oracle: price to dispute does not exist");
+
         uint256 oldPrice = priceToUpdate.price;
         priceToUpdate.price = _price;
+        priceToUpdate.timestamp = now;
```

Comment:
This has the effect of extending the dispute period each time a price is disputed, correct?

I am fine with that, but wanted to highlight it.

Reply:
Oh I forgot about that, was wondering why we didn't already include it.
Should we remove it, or add another field to indicate the time of dispute ?

Reply:
Thought @aparnakr @antoncoding ?

Reply:
Going to remove the timestamp update, and keep the timestamp relate only to the first time where price pushed.